packages/playground/src/dashboard/dao_view.vue

```diff
@@ -221,8 +221,9 @@
                 considered valid.
               </span>
               <span>
-                If the vote count is insufficient and the time limit is reached, the proposal will be rejected.</span
-              >
+                If the vote count is insufficient and the time limit is reached, the proposal will be rejected.
+              </span>
+              <a href="https://manual.grid.tf/dashboard/dao_voting/dao_voting.html" target="_blank">How to vote.</a>
```

Comment:
shouldn't we add question mark at the end ?

Reply:
added in https://github.com/threefoldtech/tfgrid-sdk-ts/pull/2027/commits/c47cad004e5be3d02c2c2e68fb04e95c56d04bf1